community/logging/src/main/java/org/neo4j/logging/FormattedLog.java

```diff
@@ -203,19 +207,30 @@ protected void writeLog( PrintWriter out, String message )
         @Override
         protected void writeLog( PrintWriter out, String message, Throwable throwable )
         {
+            requireNonNull( message, "message" );
+
             lineStart( out );
             out.write( message );
-            out.write( ' ' );
-            out.write( throwable.getMessage() );
-            out.println();
-            throwable.printStackTrace( out );
+            if ( throwable != null )
```

Comment:
This condition is already handled here: https://github.com/neo4j/neo4j/blob/2.3/community/logging/src/main/java/org/neo4j/logging/AbstractPrintWriterLogger.java#L67